plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3select/TestS3SelectPushdown.java

```diff
@@ -43,4 +102,112 @@ public void testIsCompressionCodecSupported()
         assertFalse(S3SelectPushdown.isCompressionCodecSupported(inputFormat, new Path("s3://fakeBucket/fakeObject.snappy")));
         assertTrue(S3SelectPushdown.isCompressionCodecSupported(inputFormat, new Path("s3://fakeBucket/fakeObject.bz2")));
     }
+
+    @Test
+    public void testShouldEnableSelectPushdown()
+    {
+        assertTrue(S3SelectPushdown.shouldEnablePushdownForTable(session, table, "s3://fakeBucket/fakeObject", Optional.empty()));
```

Comment:
nit: In this context it can be statically imported without decreasing readability